lua/jsonpath.lua

```diff
@@ -88,4 +88,80 @@ M.get = function()
   return path
 end
 
+local function get_yaml_path()
+  if not parsers.has_parser() then
+    return ""
+  end
+
+  local current_node = ts_utils.get_node_at_cursor()
+  if not current_node then
+    return ""
+  end
+
+  local accessors = {}
+  local node = current_node
+
+  while node do
+    local accessor = ""
+
+    if node:type() == "block_mapping_pair" then
+      local key_node = unpack(node:field("key"))
+      local key = get_node_text(key_node)
+
+      if key and starts_with_number(key) or contains_special_characters(key) then
+        accessor = string.format('["%s"]', key)
+      else
+        accessor = string.format("%s", key)
+      end
+    end
+
+    if node:type() == "block_sequence" then
```

Comment:
It could improve this here by also handling the nodes of type `flow_sequence`. I didn't find something unexpected with the following change:

```suggestion
    if node:type() == "block_sequence" or node:type() == "flow_sequence" then
```

For instance, in this line:

```yaml
json_seq: [ 3, 2, 1, "takeoff" ]
```